api/types.proto

```diff
@@ -77,6 +78,17 @@ message Resources {
 message ResourceRequirements {
 	Resources limits = 1;
 	Resources reservations = 2;
+
+	// Amount of swap in bytes - can only be used together with a memory limit
+	// -1 means unlimited
+	// a null pointer keeps the default behaviour of granting twice the memory
```

Comment:
do you mean to say "a null pointer indicates that the default behaviour of granting twice the memory is maintained" ? If so, I'd suggest clarifying